lib/get-files.js

```diff
@@ -8,11 +8,26 @@ const fileList = [
   'yarn.lock'
 ]
 
-async function getGithubFile (ghqueue, { path, owner, repo }) {
+async function getGithubFile (ghqueue, { path, owner, repo, sha }, log) {
   try {
     return await ghqueue.read(github => github.repos.getContents({ path, owner, repo }))
-  } catch (e) {
-    return { name: path, path, content: false }
+  } catch (error) {
+    // Sometimes, the requested blob (package-lock.json) is too large to fetch via the gitHub API,
+    // but we can use the Git Data API to request blobs up to 100 MB in size.
+    if (error.status === 'too_large') {
+      log.info('Could not receive contents from GitHub. File is too large', { error: error.message })
```

Comment:
maybe add something like “trying the data api for larger files now”, just so it doesn’t look like a problem in the logs